src/protocols/user_directed_commissioning/UserDirectedCommissioning.h

```diff
@@ -538,6 +538,12 @@ class DLL_EXPORT UserDirectedCommissioningClient : public TransportMgrDelegate
         mCommissionerDeclarationHandler = commissionerDeclarationHandler;
     }
 
+    /**
+     * Remove the previously set Commissioner Declaration UDC request listener.
+     *
+     */
+    void RemoveCommissionerDeclarationHandler() { mCommissionerDeclarationHandler = nullptr; }
+
```

Comment:
Do we need a special function for this, or can we just call `SetCommissionerDeclarationHandler(nullptr);`?

Reply:
right, setting to nullptr is good enough. removing.